d3ploy/demand_driven_deployment_inst.py

```diff
@@ -297,15 +330,18 @@ def calc_diff(self, commod, time):
             self.commodity_supply[commod][time] = 0.0
         supply = self.predict_supply(commod)
 
-        if self.buffer_type_dict[commod] == 'perc':
-            demand = self.predict_demand(
-                commod, time) * (1 + self.buffer_dict[commod])
-        elif self.buffer_type_dict[commod] == 'float':
-            demand = self.predict_demand(
-                commod, time) + self.buffer_dict[commod]
+        if self.buffer_type_dict[commod] == "perc":
+            demand = self.predict_demand(commod, time) * (
+                1 + self.buffer_dict[commod]
+            )
+        elif self.buffer_type_dict[commod] == "float":
+            demand = (
+                self.predict_demand(commod, time) + self.buffer_dict[commod]
+            )
         else:
             raise Exception(
-                'You can only choose perc (%) or float (double) for buffer size')
+                "You can only pick perc (%) or float (double) for buffer size"
```

Comment:
double and float are not synonyms.

Reply:
I will address this in https://github.com/arfc/d3ploy/issues/261